source/releasenotes/2024-12-10-drupal-7-d7es.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Tag1 Extended Support module installed on Drupal 7 upstream
+published_date: "2024-12-05"
+categories: [drupal, action-required]
+---
+
+The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.
+
+Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more in our [Drupal 7 LTS documentation](https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility).
+
+## Action Required
+
+To receive security ongoing security coverage for your contrib modules, apply this upstream update then enable the Tag1 D7ES module.
```

Comment:
Non-blocking but it would be nice if there were some screenshots of the module in the modules list either here in the release note or in the docs about the D7ES module.

Reply:
Should we refer to it as "the Tag1 D7ES module" or something like "the Drupal 7 Extended Support module?" I think we should go with whatever text appears in the module listing in the Drupal interfaces. 

So yeah, to @jazzsequence's point, this would be a good spot to have a screenshot of the module installation interface with the checkbox toggled to on for this module.

Reply:
FWIW it's Tag1 D7ES https://docs.tag1.com/setup/#how-to-test

Reply:
And after someone has done this step, is there something they will see or can do that verifies for them that the whole thing is working?

Reply:
If they want to fully test, they can follow the docs link from Tag1. We can include that in a docs update.

Reply:
I can spin this up and get the relevant screenshots. I'm not sure there's anything to show until there's an update to a module but the Tag1 docs do show what that looks like in practice.

Reply:
Noting that we don't link out to the Tag1 docs site for D7ES in our documentation. We should, if we don't want to rewrite 90% of it from a Pantheon context.